### ISSUE TITLE
[ENHANCEMENT] potential connections to active listening & motivational interviewing

### ISSUE THREAD

**Issue by jaanaltosaar** [interactionprinciples/interactionprinciples.org#12]
### Current Behavior

This is sometimes described as “too deep”, “too advanced” (I view this adjective as a negative thing re: accessibility). Taking a page from psychotherapy canon / law enforcement / educational canon could help.

### Expected Behavior

Accessibility and foundational skills of the type articulated in “once upon” may be implicitly assumed for users of interactionprinciples.org.

example: https://docs.google.com/document/d/17Hk7cpUOKvn3L6KxIKE7Jrs9Qn7YgLvuho49KBP8SK8/edit

however most users have not been exposed to such material (nor improv, cf sister issue)

### Steps To Reproduce

Interactions with people unfamiliar with active listening / motivational interviewing / theory of mind-demanding stances may lead to subtle power & status differentials (ditto for those fluent in the improvisational arts). 

### Anything else?

Cc @charlottemeng for viz - mind reading the document? Curious if it aligns with your experience (or not).

**Comment by charlottemeng** [comment]
The linked document has a lot to respond to, and I'll give a proper response to its contents later. To respond to the question in this issue, which is, is it harmful to employ "active listening / motivational interviewing / theory of mind" to people who do not have the same skills nor knowledge, here are my thoughts:

My approach to human connection (let's call it connection, not interaction) ties back to honoring the "presence" of another person, to treat "presence" as a gift, and that gift can only be brought forth by allowing another person to be themselves, which is made possible when the majority of people in a conversation are comfortably just being themselves. My best connections have always occurred from recognizing that a person is not a defined nor knowable entity, but rather an open question of existence that I get to witness.

I think it is quite simply, we ought to pursue the following questions in every conversation: "Am I truly seeing this person right now? Am I truly hearing this person right now?" In the process, I often employ some of the practices listed in that document (active listening, theory of mind, etc), but not with the end goal of changing me or them but to simply bear witness.

Where it has been harmful is, as noted, when the interaction is asymmetric. But not in the sense that I am the one harming, however. I've noticed--and my perception here could be wrong, since I am not an unbiased, nor objective perceiver--that I am often the one who is trying harder to "see" or "hear" the other person. This is healing for them, and extremely exhausting for me. It means I don't have a support system when I need someone, because most people are accustomed to me being the stronger source of support in our relationship.

Those are my experiences, and if you have a different perspective on what might actually be going wrong for me, let me know.

Tl;dr: feels bad and lonely when I am the one trying harder

**Comment by jaanaltosaar** [comment]
I really relate to this @charlottemeng . In relationships especially, it took me many years to learn to break up with people when I learned to build up my self esteem and confidence at assessing my emotional labor and that of my partners at the time. I felt like I had to try super hard to make it work, and often the other party did not (or I perceived it like they did not or were not "trying harder").

I also appreciate your emphasis on distinguishing interaction from _connection_. 

This is somewhat purposeful. 

* "Interaction" (Intermediate action; Mutual or reciprocal action or influence; as, the interaction of the heart and lungs on each other; https://www.websters1913.com/words/Interaction) feels clinical and at a remove.
*   _connection_ (The act of connecting, or the state of being connected; junction; union; alliance; relationship. That which connects or joins together; bond; tie. A relation; esp. a person connected with another by marriage rather than by blood; -- used in a loose and indefinite, and sometimes a comprehensive, sense. The persons or things that are connected; as, a business connection; the Methodist connection.) feels much too dominant of a stance or overloaded, over-indexing on the potentialities of what might be anything from a glance on the street to an oncoming passerby or a three-hour heart2heart sitting next to someone on a long bus ride...

I perhaps prioritize the disaffected, clinical remove in the word "interaction" in a similar way that this (political) post highlights the columbiafacts.org diagram: https://adamtooze.substack.com/p/chartbook-279-columbia-universitys. I'm not sure what I make of it, whether it is true, other than a professor named Adam decided to publish this and wanted to help his students, some of whom may now be in jail. The "interactions" in such diagrams or power mapping exercises feel more accurate than the "connections" between entities... 

(Working this out as we speak. Feel free to push back on any and all of the above; syncretism can only be wrought through the healthy dialectic we may be attempting to hew here  :)

**Comment by charlottemeng** [comment]
I think your analysis of the differences between connection and interaction is quite accurate, and for the purposes of this project, "interaction principles" is more appropriate than "connection principles" partly because power is a core theme among the principles. 

I think I invoked "connection" because "active listening" is very much an attempt at connection and not just an interaction where two entities are trying to give each other information (and the value/success of their information exchange is dependent on things such as power differences, mutual respect, etc, which inform the biases embedded in the presentation of the information that each side gives). I think "interaction" applies to conversations between individuals where both individuals hope to influence each other's decisions and thought processes towards a positive direction; "connection" implies that there is emotional labor occurring (eg active listening is quite emotionally laborious), and an attempt to generate positive emotions between both parties in a conversation. 

So I agree that interaction is clinical and connection is emotionally loaded. But with the proposal to incorporate active listening somehow into the interaction principles, I think these are not merely "interaction" principles but perhaps "Being-with-others principles", to use Heidegger's odd terminology.

These are also just my rough thoughts and rough intuitions. I probably worded something incorrectly and likewise, appreciate and encourage pushback here.
